docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -557,6 +557,23 @@ spec:
 > **Note:** The IPv6 address for the VM and default gateway **must** be the ones
 > shown above.
 
+### masquerade - IPv6 single-stack support
```

Comment:
I think this needs to be `####` (4), no?

Reply:
done